HiP-DataStore.Model/Entity/Rating.cs

```diff
@@ -0,0 +1,32 @@
+﻿using PaderbornUniversity.SILab.Hip.DataStore.Model.Events;
+using System;
+
+namespace PaderbornUniversity.SILab.Hip.DataStore.Model.Entity
+{
+    public class Rating
```

Comment:
On the other hand... a cleaner Command & Query separation could be achieved if we query the database in `ExhibitsController.GetRating(int)` (instead of using `RatingIndex`). What do you think?